src/main/java/mate/academy/springbootwebgreqit/model/Book.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.springbootwebgreqit.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true where id = ?")
+@Where(clause = "is_deleted = false")
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(length = 100)
+    private String title;
+    @Column(length = 50)
+    private String author;
+    @Column(unique = true, length = 13)
+    private String isbn;
+    @Column(precision = 10, scale = 2)
```

Comment:
Add constraint for all required fields


FYI This should be done in previous HWs